<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>526</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>794</width>
    <height>526</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>794</width>
    <height>526</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>701</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="unlockPage">
    <widget class="QLineEdit" name="inputPasswordTextBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="unlockButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Unlock</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>531</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter your Master Password to unlock your Vault</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toCreateButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>380</y>
       <width>181</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Create new vault</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showUnlockButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>180</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>show</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="createPage">
    <widget class="QLineEdit" name="createPasswordTextBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>20</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="createButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>531</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter your Master Password to create your Vault</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toUnlockButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showCreateButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>180</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>show</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
